PhotoMap/Features/Category/Controllers/CategoryViewController.swift

```diff
@@ -9,4 +9,68 @@ import UIKit
 
 class CategoryViewController: BaseViewController {
     
+    // MARK: - Variables
+    private var viewModel: CategoryViewModelType?
+    private var cancelBag = CancelBag()
+    
+    // MARK: - @IBOutlets
+    @IBOutlet private weak var tableView: UITableView!
+    private lazy var doneButton = UIBarButtonItem(title: L10n.Categories.NavigationItem.RightButtonItem.done,
+                                                  style: .done, target: self, action: #selector(doneButtonPressed))
```

Comment:
We can also replace this with UIBarButton publisher that we have in UIBarButtonItem extensions.
So you could do the following:
```
doneButton.publisher
      .subscribe(viewModel.doneButtonSubject)
      .store(in: cancelBag)
```